xls/dslx/run_routines/run_routines.h

```diff
@@ -33,8 +33,8 @@
 #include "absl/status/statusor.h"
 #include "absl/time/time.h"
 #include "absl/types/span.h"
+#include "re2/re2.h"
 #include "xls/dslx/bytecode/bytecode_interpreter_options.h"
-#include "xls/dslx/default_dslx_stdlib_path.h"
```

Comment:
If  this header is now removed, we probably don't need to link `default_dslx_stdlib_path` anymore.

```
bant dwyu //xls/dslx/run_routines:run_routines
```

Reply:
Done -- were you planning to make this part of presubmit?

Reply:
One of the next times I am build-gardener ...